inc/itilfollowup.class.php

```diff
@@ -535,6 +535,14 @@ function rawSearchOptions() {
          'right'              => 'all'
       ];
 
+      $tab[] = [
+         'id'                 => '6',
+         'table'              => $this->getTable(),
+         'field'              => 'itemtype',
+         'name'               => __('Request source'),
+         'datatype'           => 'dropdown'
+      ];
+
```

Comment:
New searchOption to search ITILFollowups by type (needed for /search/TicketFollowup).